docker-compose.prod.yml

```
@@ -202,7 +207,7 @@ services:
   ed2:
     user: "${UID:-1001}:${GID:-1001}"
     image: pecan/model-ed2-2.2.0:${PECAN_VERSION:-latest}
-    restart: unless-stopped
+    platform: linux/amd64
```

We don't need to remote the `restart: unless-stopped` tag as this will auto shut down the container if somehow the container has stopped. Can you add the abpve line too?
```suggestion
    restart: unless-stopped
    platform: linux/amd64
```

---

yes you are right.